assembl/static2/webpack.config.js

```diff
@@ -54,5 +84,6 @@ module.exports = {
           }
         }),
         new webpack.optimize.UglifyJsPlugin(),
+        new ExtractTextPlugin("[name].css"),
```

Comment:
No, we never really thought that the hot-module replacement will be affected by the themefication. But at the very least, in the documentation, should add that and theme changes must be recompiled. It's a one-time processes...I hope.